ApiBundle/Controller/ContentController.php

```diff
@@ -113,6 +113,12 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.content');
+
+        if ($request->get('entityId') && $request->get('language') && $request->get('version')) {
```

Comment:
It's ok for me and i did it. But there's still a problem with contentController. I call shoOrCreate method which return a facade. How to change it in collection facade? @nicolasThal